pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -722,8 +723,10 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 	}
 
 	// Take SMBios values from the VirtualMachineOptions
-	domain.Spec.OS.SMBios = &SMBios{
-		Mode: "sysinfo",
+	if runtime.GOARCH != "ppc64le" {
```

Comment:
it looks like some ppc64le related hacks, can you add a comment, why we do not enable it on ppc64le architecture.

Reply:
Ok, will do